R/geom-rug.r

```diff
@@ -43,11 +40,17 @@
 #'    coord_cartesian(clip = "off") +
 #'    theme(plot.margin = margin(1, 1, 1, 1, "cm"))
 #'
+#' # increase the line length and
+#' # expand axis to avoid overplotting
+#' p + geom_rug(length = unit(0.05,"npc")) +
+#'    scale_y_continuous(expand=c(0.1,0.1))
```

Comment:
Could you follow [the tidyverse style guide](https://style.tidyverse.org/syntax.html#spacing)? You need more spaces here.